docs/mine/spacerace.md

```diff
@@ -79,6 +79,40 @@ Yes, you can combine your competition results from multiple miners. Once the com
 #### How are rewards distributed?
 If you’re eligible for rewards, someone from CoinList will reach out to your provided email address shortly after the competition to conduct AML/KYC and coordinate delivery of the tokens. You will have the option to receive rewards directly to your wallet.
 
+#### How do I prioritize deals from the competition bots?
+By default, Lotus nodes accept all inbound deals that match their criteria. 
+However, during the Space Race competition, miners may want to limit the clients that 
+they accept deals from, so to avoid any malicious agents creating spam deals. 
+To do this, modify the `~/.lotusminer/config.toml` file to include a `Filter` param.
+This param should be a shell command that will be run when processing a deal proposal. 
+
+```
+~/.lotusminer/config.toml
+
+[Dealmaking]
+Filter = <shell command>
+```
+
+The deal info is piped into stdin as JSON. Deals are accepted if the `Filter`'s
+ exit code is 0, for any other exit code they will be rejected. Examples:
```

Comment:
Might be worth linking to a gist with example json or something like that. This allows for implementing much more advanced filtering than 'only get deals from this client'.

Reply:
Good idea (and thanks for providing the JSON in hastebin). Punting to another issue though since it belongs outside of Space Race page.